src/main/resources/schema/schema_v9.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE system_table
+  ADD COLUMN langtags TEXT ARRAY;
+
+CREATE TABLE system_settings (
+  key   VARCHAR(255) PRIMARY KEY,
+  value TEXT
+);
+
+INSERT INTO system_settings (key, value) VALUES ('langtags', $$["de-DE", "en-GB"]$$);
```

Comment:
Single language, `de-DE` if we stay with the current demo data and `en-US` if we decide to switch to English (I don't want to write `colour` ;)).